config_default.yaml

```diff
@@ -87,4 +87,8 @@ teracy-dev-k8s:
     host_vars:
       ingress_nginx_enabled: "True"
       helm_enabled: "True"
-      helm_version: v2.11.0
+      helm_version: v2.12.1
+      disable_ipv6_dns: "True"
```

Comment:
it's better to move these stuff to teracy-dev-k8s extension instead.

Reply:
@hoatle I applied your comment. Please check again at https://github.com/teracyhq-incubator/teracy-dev-k8s/pull/72! Thank you!